paddle/fluid/pybind/pybind.cc

```diff
@@ -414,8 +414,26 @@ All parameter, weight, gradient are variables in Paddle.
              self.set_falsenet(net.Clone());
            });
 
+  py::class_<ExecutorPrepareContext>(m, "ExecutorPrepareContext")
+      .def("__init__", [](ExecutorPrepareContext &instance, ProgramDesc &desc,
```

Comment:
It seems that `__init__` method  is not needed

Reply:
removed